ocaml/utils/import_info.ml

```diff
@@ -14,53 +14,65 @@
 
 module CU = Compilation_unit
 
-type t =
-  | Normal of CU.t * Digest.t
-  | Normal_no_crc of CU.t
-  | Other of CU.Name.t * (CU.t * Digest.t) option
+module Intf = struct
+  type t =
+    | Normal of CU.Name.t * Digest.t (* Unpacked, so compilation unit = name *)
+    | Name_only of CU.Name.t
+    | Other of CU.Name.t * (CU.t * Digest.t) option
 
-(* CR xclerc: Maybe introduce Other_no_crc to flatten the option *)
+  let create cu_name ~crc_with_unit =
+    match crc_with_unit with
+    | None -> Other (cu_name, None)
+    | Some (cu, crc) ->
+      (* For the moment be conservative and only use the [Normal] constructor
```

Comment:
This comment seems out of date now, but maybe there should still be a comment here about the decision based on the pack prefix.

Reply:
Updating the comment.